src/components/Header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Header.module.css";
+
+
+
+const Header = () => {
+    return (
+        <header className = {s.header}>
+        <img src="https://i.pinimg.com/originals/d9/12/09/d91209340bdc005936c46323a62caaff.png"/>
```

Comment:
Consider putting this string url into a constant variable in the current file and using the variable reference here.

Reply:
complite